sdk/python/endpoints/batch/text-summarization-batch.ipynb

```diff
@@ -0,0 +1,1006 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "# Using batch deployments for NLP processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "Importing the required libraries. This notebook requires:\n",
+    "\n",
+    "- `azure-ai-ml`\n",
+    "- `mlflow`\n",
+    "- `azureml-mlflow`\n",
+    "- `numpy`\n",
+    "- `pandas`\n",
+    "- `huggingface`\n",
+    "- `torch`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 153,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Class SystemCreatedStorageAccount: This is an experimental class, and may change at any time. Please see https://aka.ms/azuremlexperimental for more information.\n",
+      "Class SystemCreatedAcrAccount: This is an experimental class, and may change at any time. Please see https://aka.ms/azuremlexperimental for more information.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from azure.ai.ml import MLClient, Input\n",
+    "from azure.ai.ml.entities import (\n",
+    "    BatchEndpoint,\n",
+    "    BatchDeployment,\n",
+    "    Model,\n",
+    "    AmlCompute,\n",
+    "    Data,\n",
+    "    BatchRetrySettings,\n",
+    "    CodeConfiguration,\n",
+    "    Environment,\n",
+    ")\n",
+    "from azure.ai.ml.constants import AssetTypes, BatchDeploymentOutputAction\n",
+    "from azure.identity import DefaultAzureCredential"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "## Accessing the Azure Machine Learning workspace"
```

Comment:
Some cells have periods, others don't..! I think we should probably standardize on ponctuations :)

Reply:
On titles?

Reply:
Oh I see - yeah nevermind